foosball/games/models.py

```diff
@@ -25,7 +24,7 @@ class Game(TimeStampedModel):
     )
 
     played_at = models.DateTimeField(
-        default=datetime.datetime.now,
+        default=timezone.now,
```

Comment:
`timezone.now` won't return the current time on the current timezone if that was the purpose of this. I think you need something like `timezone.localtime(timezone.now())` to get the actual local current time. :smiley: